traits/testing/optional_dependencies.py

```diff
@@ -45,9 +45,9 @@ def optional_import(name):
 numpy = optional_import("numpy")
 requires_numpy = unittest.skipIf(numpy is None, "NumPy not available")
 
-numpy_testing = optional_import("numpy.testing")
```

Comment:
I hope other ETS packages aren't using `numpy_testing`

Reply:
it looks like this code also lives in #1710

Reply:
Yes, I wanted to see this PR passing, so I merged in the fix from #1710. I've now merged the main branch in, so this PR should just contain the setup.py changes again.